app/services/doc_auth/acuant/config.rb

```diff
@@ -0,0 +1,33 @@
+module DocAuth
+  module Acuant
+    # @!attribute [rw] exception_notifier
+    #   @return [Proc] should be a proc that accepts an Exception and an optional context hash
+    #   @example
+    #      config.exception_notifier.call(RuntimeError.new("oh no"), attempt_count: 1)
+    Config = RedactedStruct.new(
+      :assure_id_password,
+      :assure_id_subscription_id,
+      :assure_id_url,
+      :assure_id_username,
+      :facial_match_url,
+      :passlive_url,
+      :timeout,
+      :dpi_threshold,
+      :sharpness_threshold,
+      :glare_threshold,
+      :exception_notifier,
+      :warn_notifier,
+      keyword_init: true,
+      allowed_members: [
+        :assure_id_subscription_id,
+        :assure_id_url,
+        :facial_match_url,
+        :passlive_url,
+        :timeout,
+        :dpi_threshold,
+        :sharpness_threshold,
+        :glare_threshold,
```

Comment:
I forgot to add `warn_notifier` and `exception_notifier` to the allowed/nonredacted members, can we add them here? if not I can follow up in another PR

Reply:
Done in b7b535a